theatre/studio/src/Studio.ts

```diff
@@ -217,6 +220,112 @@ export class Studio {
     this._store.redo()
   }
 
+  pasteKeyframes({
+    sheetObject,
+    trackId: selectedTrackId,
+    keyframes: keyframesToPaste,
+    position,
+  }: {
+    sheetObject: SheetObject
+    trackId: string
+    keyframes: CopiedKeyframes[]
+    position: number
+  }) {
+    const {address, sheet} = sheetObject
+    const allTracks = val(sheetObject.template.getArrayOfValidSequenceTracks())
+    const selectedTrackIndex = allTracks.findIndex(
+      ({trackId}) => trackId === selectedTrackId,
+    )
+
+    /**
```

Comment:
Tiny nitpick, let's only use JSDoc strings for actual documentation. For long multiline comments, you can either do
```js
/*
 * bla
 * bla
 */

or

// bla
// bla
```

I prefer to use asterisk too to make long multiline comments more obviously comments, but by omitting the second leading asterisk, it's not JSDoc. For more on my preferred style of commenting: https://google.github.io/styleguide/jsguide.html#formatting-comments